### DISCUSSION TITLE
Expedition 12: Omega not working

### DISCUSSION THREAD

**Discussion by Dcmasda** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#19] [General]
Don't know if I've done something wrong, but as the latest update (worlds pt1) instead of displaying omega expedition, it shows current liquidators expedition with 400+ months remaining

**Comment by cwmonkey** [comment]
Looks fine to me. I used my version from https://cwmonkey.github.io/nms-expeditions/ and did a diff against the one in this repo and they look the same. Possible you grabbed the wrong json file?

![image](https://github.com/user-attachments/assets/a6bff1a9-3a71-40d7-a195-9d7ef2778ed1)
![image](https://github.com/user-attachments/assets/0adf2add-e40d-4a46-96a5-06ad76606218)

**Reply by Dcmasda** [reply]
Will try, tysm for the quick reply!
